client/my-sites/plans-features-main/index.jsx

```diff
@@ -483,6 +546,7 @@ export class PlansFeaturesMain extends Component {
 				<QueryPlans />
 				<QuerySites siteId={ siteId } />
 				<QuerySitePlans siteId={ siteId } />
+				<QuerySitePurchases siteId={ siteId } />
```

Comment:
Right now purchases will only be used for Jetpack sites. Should we render this query component only in the instance where `this. isJetpackBackupAvailable()` is `true` or something similar?

Will be redundant anyway if we decide to move `getProductsSelectorSubHeader` to a component as I suggested above.